src/App.tsx

```diff
@@ -3,15 +3,17 @@ import React from "react";
 import { Provider } from "react-redux";
 import { store } from "./store/store";
 import styled from "styled-components";
-
+import TranslationProvider from "./components/TranslationProvider";
```

Comment:
Something is wrong with your formatter configuration.

please fix the formatting

Reply:
ESlint was added